rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/util/CovidRDTJsonFormUtils.java

```diff
@@ -153,7 +152,8 @@ protected void prePopulateRDTPatientFields(Patient patient, JSONObject field) th
         super.prePopulateRDTPatientFields(patient, field);
         String key = field.getString(JsonFormUtils.KEY);
         if (PATIENT_SEX.equals(key)) {
-            field.put(JsonFormUtils.VALUE, patient.getPatientSex().toLowerCase());
+            String translatedSex = patient.getPatientSex();
+            field.put(JsonFormUtils.VALUE, "Perempuan".equalsIgnoreCase(translatedSex) ? "female" : "male");
```

Comment:
Why are we doing this?

Reply:
actually, we are using gender in relevance that is working only English locale not id for that reason i just did that https://github.com/onaio/rdt-standard/blob/b7588b844a24415d2d529b5c0a7036880ddcf3b7/rdt-open-reader/src/covidIndonesia/assets/rule/patient-diagnostics-form-relevance-rules.yml#L12

Reply:
the key should not change based on locale

Reply:
it's a value like if we change the locale `en` to `id` then the patient sex change `female` to `Perempuan` that's why the relevance condition is not working correctly because in the relevance condition we are using harcoded `female` value

Reply:
If that is the case, we should use a string.xml value to pick the correct translation based on locale.

Reply:
feedback implemented

Reply:
I think what I meant was instead of:

`field.put(JsonFormUtils.VALUE, "Perempuan".equalsIgnoreCase(translatedSex) ? "female" : "male")`

have something like:

`field.put(JsonFormUtils.VALUE, context.getString(R.string.female).equalsIgnoreCase(translatedSex) ? "female" : "male")`.  

we get the value from strings.xml and not hardcode `"Perempuan"`.

Reply:
done